src/site/content/en/metrics/inp/index.md

```diff
@@ -44,18 +44,72 @@ This article explains how INP works, how to measure it, and offers advice for im
 
 ## What is INP?
 
-INP aims to represent a page's overall responsiveness by measuring all click, tap, and keyboard interactions made with a page. The longest of those observed interactions&mdash;with some exceptions noted below&mdash;is chosen as the page's INP value when the user is done with the page.
+INP aims to represent a page's overall responsiveness. It does so by measuring every click, tap, and keyboard interaction made with a page. An _interaction_ is one or more event handlers that fire during the same logical user gesture. For example, "tap" interactions on a touchscreen device include multiple events, such as `pointerup`, `pointerdown`, and `click`. An interaction can be driven by JavaScript, CSS, built-in browser controls (such as form elements), or a combination thereof.
+
+The duration for every interaction is captured by the [Event Timing API](https://caniuse.com/mdn-api_performanceeventtiming). In cases where there are few interactions made with a page&mdash;less than 50 to be exact&mdash;the interaction with the highest latency of all observed interactions is chosen as the page's INP when the user is done with the page.
+
+However, in cases where there are many interactions with a page&mdash;50 or more to be specific&mdash;the interaction with the highest latency isn't the page's INP, but rather a high-latency interaction that's close to the worst. The table below provides more information on how INP is calculated based on the total number of interactions made with a page.
```

Comment:
This is confusing to me. Why would be exclude events when we have them? I still don't get why we take 98% when we can.

Can we explain the reasoning/thought process behind this, instead of just giving the table to show _how_ we do this calc, but not _why_ we do this at all?

Reply:
Why do we take the 98th percentile* for INP? That to me is not explained. This change explains how the 98th percentile is calced, but not _why_ we bother doing this?

Is it something to do with excluding the extremes? If so why 98th percentile? Why not 90th? or 75th?

`*` And btw I do agree with removing "98th percentile" phrase itself - too confusing with 75th percentile. But still should explain WHY we do this.

Something like:

> For pages with many interactions, we often saw outlier INP values that were unrepresentative of how responsive the page felt to users overall. For this reason we exclude the very worst interaction values for these pages to get a more accurate representation of responsiveness on these pages. The INP value used is logged in the table below.

Or something like that.

WDYT @malchata / @anniesullie / @mmocny ?